Orange-Skeletons/welcome-to-nashville
# Jg link parks

# Description

Added the function on parks to show on results, clear, and save.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-link-parks`
4. entering 'dog_park' in the park search field. (hopefully soon to be a drop down)

# Checklist:
